cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/CloudStack.java

```diff
@@ -13,6 +13,8 @@
  */
 public class CloudStack {
 
+    public static final String CLOUDWATCH_CREATE_PARAMETER = "createCloudWatchAlarm";
```

Comment:
I think this would be better in `PlatformParametersConsts`

Reply:
moved